react-client/src/Components/TextInput.js

```diff
@@ -0,0 +1,48 @@
+import React, { Component } from 'react';
+
+const inputStyle = {
+    width: '25vw',
+    padding:'0.5vw', 
+    fontSize: 'calc(10px + 2vmin)', 
+    textAlign:'center',
+    margin:'1vh'
+}
+
+const submitStyle = {
+    width:'15vw',
+    fontSize: 'calc(10px + 2vmin)',
+    textAlign:'center',
+}
+
+
+
+class TextInput extends Component {
+    constructor(props) {
+        super(props);
+        this.state = {value:''}
+        this.handleChange = this.handleChange.bind(this);
+        
```

Comment:
The handleSubmit function doesn't seem to have to have 'this' defined, you likely need to bind it here like you did with the handleChange function